src/main/java/core/basesyntax/repository/FruitDB.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.repository;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+public class FruitDB {
+    public static final Map<String, Integer> fruitsOnStock = new LinkedHashMap<>();
+}
```

Comment:
We can use just HashMap
Because we have fruit like a key, it will be always change quantity depends about operation.
But its only my opinion, because I don't understand why we need linkedhashmap  -_-